src/aat/resources/features/F-032/S-124.td.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "must retrieve printable documents successfully for correct inputs",
+
+  "_guid_": "S-124",
+  "_extends_": "F-032_Base_Test_Data",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains the case data of the case just created",
+    "contains a link to the printable documents that were uploaded to the case just created, along with a HTTP 200 OK"
+  ],
+
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Content-Encoding": "gzip",
+      "Content-Type": "application/json;charset=UTF-8",
+      "Content-Length": "[[ANY]]"
```

Comment:
Lines 16-18, need to move to F-032_Case_Creation_Data_With_Document.td.json

Reply:
why? the case creation data has nothing to do with the test data?

Reply:
do you mean move them to the F-032_Base_Test_Data file? but these headers are only applicable to the happy path scenario, not the other scenarios...